atlp-rwanda/elite-bn-be
# ft(user-roles): super admin should set user role via their emails

#### What does this PR do?
This PR help in setting user's role by email
super administrator can give different user a role via their emails 

#### How should this be manually tested?

- After cloning this PR make sure you create a `.env` file by following the `.env.example` file in this repo
- Run `npm run migration` 
- Run `npm run seed ` for creating a super admin, use email: kikolulu@gmail.com & password: kikolulu@123

To get all user's information in case you want to get their emails use  `GET: api/v1/user/alluser ` 
Then by using postman you can use `POST: api/v1/user/role` and set the body of the following to assign role to user
which can be "requester", "travel admin", "accommodation supplier", "manager" and "super admin"

               "userEmail": "email",  
                "userRole": "role"

#### Any background context you want to provide?
-  Please make sure you are using the same information from`.env` file as the one from login endpoint like  `JWT_SECRET `....
- Make sure your database is connected well

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/182201568
